spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/method/annotation/ResponseBodyEmitterReturnValueHandler.java

```diff
@@ -209,7 +211,7 @@ public void complete() {
 
 		@Override
 		public void completeWithError(Throwable failure) {
-			this.deferredResult.setErrorResult(failure);
+			// no-op
 		}
```

Comment:
Couldn't we leave this as it was before? It is true that the PR removed the internal use of the method within the try-catch of `ResponseBodyEmitter#sendInternal` and that any errors from the container side should work as expected but a controller could also call `completeWithError` and the change eliminates that option (and would cause a regression).

If a container and an application thread both try to `setErrorResult` I suppose only one could win?